python/main.py

```diff
@@ -18,28 +31,59 @@
     allow_headers=["*"],
 )
 
+DATABASE_URL = "../db/mercari.sqlite3"
```

Comment:
must: Dockerfileの変更に基づいて、こちらのパスも変更する必要があります
```suggestion
DATABASE_URL = "./db/mercari.sqlite3"
```